sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/components/AppView.groovy

```diff
@@ -45,6 +48,17 @@ class AppView extends VerticalLayout {
         this.addComponents(titleLabel, createSpacer(2, Unit.EM), hotbar, projectOverviewView, sampleOverviewView)
     }
 
+    private void addProjectSelectionListener(){
```

Comment:
NIT: As you are not adding a listener as such but you listen directly maybe an alternative name for this method would be `listenToProjectSelectionChange`

Reply:
You are right, I like that suggestion